wiki/tests/test_template_filters.py

```diff
@@ -145,12 +145,12 @@ def test_strip_tags(self):
 
         content = """
         <h1>Some dummy</h1> text. <div>Actually</div> I don't what to write,
-        heh. Don't now, <b>maybe</b> I should citate <>Shakespeare<> or Byron.
```

Comment:
I'm not entirely sure what the original intention was. Having `<>` in some HTML just seems incorrect - if you want a literal `<` it should be written `&lt;`, and that's what you'd have if you entered it in an editor.

I therefore thought it was understandable that the behaviour of Django's `striptags` has changed, and probably django-wiki shouldn't have a test for this specific corner case. Don't know.